### PR TITLE
Add Spanish Documentation

### PR DESCRIPTION
**Pull request by Carlos-Carsdfj**
Adding the introduction , quick start and  code.json in spanish lenguage

### CODE REVIEW

**Comment by Nojipiz** [comment]
This is awesome! 
I will commit if this PR is accepted :D

**Comment by Carlos-Carsdfj** [comment]
Thank you, as soon as I have time I will try to translate what I can.

File: i18n/es/docusaurus-plugin-content-docs/02-quick-start.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Formateo y Linting
+
+Formateo y Linting no son compatibles con algunos LSP de forma predeterminada.
+Esto deben instalarse / configurarse por separados.
+
+Ver [lenguajes](./languages/README.md) donde se puede abordar cada idioma con su formateo y rodado.
+
+## Fuentes Nerd
+
+Installing a [nerd font](https://www.nerdfonts.com/) is recommended. Otherwise some symbols won't be rendered properly. For more information go to the [configuration section ](./configuration/04-nerd-fonts.md).
```
Comment (vonPB) [inline i18n/es/docusaurus-plugin-content-docs/02-quick-start.md:41]: you probably meant to remove this :)

**Review by vonPB — CHANGES_REQUESTED** [review]

File: i18n/es/docusaurus-plugin-content-docs/02-quick-start.md (hunk 1 above)
Comment (Carlos-Carsdfj) [reply i18n/es/docusaurus-plugin-content-docs/02-quick-start.md:41]: I'm going to change it, sorry!

Comment (vonPB) [reply i18n/es/docusaurus-plugin-content-docs/02-quick-start.md:41]: no worries at all!

**Review by vonPB — APPROVED** [review]

**Pull request merged**